internal/config/config.go

```diff
@@ -149,6 +149,12 @@ func (c *Config) configArchive() {
 			s3.Port = 443
 		}
 
+		if viper.IsSet("archive.region") {
+			s3.Region = viper.GetString("archive.region")
+		} else {
+			s3.Region = "us-east-1"
+		}
```

Comment:
I'd really prefer having defaults initializations before (collected), second best would be the go pattern

`if s3.Region = "us-east-1"; viper.IsSet("archive.region") {
￼			s3.Region = viper.GetString("archive.region")
￼		}

Reply:
> Can we also join the main parts of configInbox and configArchive in and just pass the prefix to pick settings from (in internal/config/config.go) to a common function?

Changes to the config module should be done a a separate PR, but yes I see no reason not to.